app/api/volunteers/entities.rb

```diff
@@ -12,12 +12,16 @@ class HelpSeeker < Common::Entities::User
     class Need < Common::Entities::Need
       expose :added_by, using: HelpSeeker, expose_nil: true
       expose :reviews, using: Common::Entities::Review, expose_nil: true
+      expose :status_updated_at
+      expose :updated_by
     end
 
     class RecommendedNeed < Need
       expose :contact_info
       expose :contact_phone_number
       expose :chosen_by, using: Volunteer, expose_nil: true
+      expose :status_updated_at
+      expose :updated_by
```

Comment:
Being the child class of `Need` it inherits all its fields, there's no need to expose them again, just expose those fields that are not part of the parent class, as is the case for contact information.